### PR TITLE
Readme ci status

### PR DESCRIPTION
**Pull request by roymacdonald**
I added to the readme CI status badges for each platform. I think this is super useful and handy.
@arturoc @bakercp @ofZach @ofTheo 

(see it in action here https://github.com/roymacdonald/openFrameworks/tree/readmeCIStatus -- appveyor needs a page refresh for some reason)

### CODE REVIEW

**Comment by ofTheo** [comment]
This looks great to me! Nice to quickly see what is working/broken.

**Comment by bakercp** [comment]
This seems good. Would be nice if we could get both master and stable branches listed.

**Comment by roymacdonald** [comment]
@bakercp please correct me if wrong. The stable branch is the one with the build distributed at openframeworks.cc/download ?

I made what you suggested as a different branch, simply to check and if the rest agrees I can merge it into this PR
https://github.com/roymacdonald/openFrameworks/tree/readmeCIStatus2
cheers

**Comment by arturoc** [comment]
yes that's it, stable should be the same as the current download, and yes just add those changes to this PR and i'll merge them

**Comment by roymacdonald** [comment]
@arturoc Just added the stable and master status. ready for merging

File: README.md [hunk 1]
```diff
@@ -7,21 +7,24 @@ openFrameworks is a C++ toolkit for creative coding.  If you are new to OF, welc
...
-iOS                          | [![iOS Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="ios"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-tvos                         | [![tvos Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="tvos"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-Android Arm7                 | [![Android Arm7 Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=GRADLE_TARGET="compileArm7DebugSources"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-Android X86                  | [![Android X86 Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=GRADLE_TARGET="compileX86DebugSources"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
+* **Master** branch has the newest, most recently updated code
```
Comment (bakercp) [inline README.md]: - The **master** branch contains the newest, most recently updated code. This code is packaged and available for download in the "Nightly Builds" section of [openframeworks.cc/download](https://openframeworks.cc/download/).
- The **stable** branch contains the code corresponding to the last stable openFrameworks release. This stable code is packaged and available for download at [openframeworks.cc/download](https://openframeworks.cc/download/).

Comment (roymacdonald) [reply README.md]: super. thanks. Just added this clarification.

**Comment by bakercp** [comment]
Looks great.

**Comment by bakercp** [comment]
@roymacdonald Thanks :)

**Pull request merged**